lib/core/public/run-rules.js

```diff
@@ -18,6 +19,7 @@ function cleanup() {
 function runRules(context, options, resolve, reject) {
 	'use strict';
 	try {
+		// axe._nodeMap = new WeakMap();
```

Comment:
remove commented code

Reply:
This is an outdated file (first iteration of the pr)